crates/nu-cli/Cargo.toml

```diff
@@ -98,6 +99,12 @@ trash = {version = "1.0.1", optional = true}
 [target.'cfg(unix)'.dependencies]
 users = "0.10.0"
 
+# TODO this will be possible with new dependency resolver
+#   (currently on nightly behind -Zfeatures=itarget):
+#   https://github.com/rust-lang/cargo/issues/7914
+#[target.'cfg(not(windows))'.dependencies]
```

Comment:
Unfortunately, num-format needs Clang to format per system locale on Windows, which seems disproportionate for this functionality alone. num-format puts system locale behind a feature flag in Cargo.toml - currently, Cargo's dependency resolver can't differentiate features by system. Fortunately, there's a new dependency resolver (#https://github.com/rust-lang/cargo/issues/7914) which will enable this should it reach stable.

Reply:
Ya, I noticed your TODO comments. What does that mean for this functionality now? We can't compile without clang? Also, I noticed the CI failed but that looks like a bogus error. Not sure what's going on there.

Reply:
Right now it just uses `en`, so it works but won't actually localize - with those lines uncommented it wasn't compiling on the Windows azure jobs.

Reply:
ok, thanks.

Reply:
This seems good for now. Once it lands, we should file an issue for follow-up so we can track the work on cargo.